content/sensu-core/2.0/reference/rbac.md

```diff
@@ -10,117 +10,200 @@ menu:
     parent: reference
```

Comment:
Can you switch `platformContent` to `false`?

Reply:
Sure. That brings up a question that's been on the back of my mind - do we have a template for what should be in the front matter? I've been using hugo to generate and adding in the menu tree stuff from other docs, but haven't been really paying attention to what *should* be there at a minimum or what those attributes do 😳

Reply:
I know we have these templates now for Sensu 1.0: https://github.com/sensu/sensu-docs-site/blob/master/content/sensu-core/0.29/files/sensu-doc-template.md

But more specifically, `platformContent` is used to generate a dropdown button to switch the platform (i.e., rhel, ubuntu, windows) used so we can create sections specific to each platform. The [installation](https://github.com/sensu/sensu-docs-site/blob/master/content/sensu-core/2.0/getting-started/installation-and-configuration.md) guide is a good example. I don't believe we have any documentation on that though.

Reply:
Oh gotcha, that makes total sense.